### ISSUE TITLE
Properties in a property drawer at the start of a file

### ISSUE THREAD

**Issue by beriterb** [kaushalmodi/ox-hugo#685]
It seems `HUGO_DRAFT` property in property drawer at the start of a file does not get picked up. `#+hugo_draft` works fine.

## Ox-Hugo Debug Information

<details>
  <summary>Debug Info</summary>

  

</details>

**Comment by kaushalmodi** [comment]
> It seems HUGO_DRAFT property in property drawer at the start of a file does not get picked up. #+hugo_draft works fine.

That's correct. The property drawers are parsed only for the subtree based export flow. (The top-of-the-file `ID` property is a special case though, and that will work as expected.)

In the example you shared, you are using the file based flow, so the `#+hugo_draft` keyword will need to be used. 

Corrected version:

```org
:PROPERTIES:
:ID:       7609e428-8568-4fa6-8c3c-99da28510d49
:END:

#+title: ARM Cortex
#+hugo_categories: ok
#+hugo_draft: true

* CMSIS
...
```

**Comment by beriterb** [comment]
I see. Just found some solid utility functions in Org-roam to work with `#+xxx`-like properties.
Thanks for replying. I'll close for now: )
